### PR TITLE
fixed filepath bug

### PR DESCRIPTION
**Pull request by ShiboSoftwareDev**
there are two new issues:
- api\tests\routes\export_requests\list.test.ts timed out but works if you increase timeout to 10k ms
- I can't export anything maybe I think the problem is in the file: run-entrypoint-file.ts

### CODE REVIEW

**Comment by seveibar** [comment]
We can increase the timeout for the test inside the workflow (not a big deal) do you want to make that edit to see if the tests pass? I didn't understand your other issue, the diff is a bit hard to read because of a formatting issue, maybe you could run `bun run format`

File: cli/lib/soupify/run-entrypoint-file.ts [hunk 1]
```diff
@@ -17,44 +17,49 @@ export const runEntrypointFile = async (
...
+    debug(`starting process....`)
 
-  const rawSoup = await readFile(tmpOutputPath, "utf-8")
-  const errText = processResult.stderr
+    try {
+      const processResult = await processCmdPart1
+        .stderr(debug.enabled ? "inheritPiped" : "piped")
+        .stdout(debug.enabled ? "inheritPiped" : "piped")
+    } catch (e) {
+      console.log(e)
```
Comment (seveibar) [inline cli/lib/soupify/run-entrypoint-file.ts]: hmm we should probably revert this part because it allows continuing after errors

Comment (seveibar) [reply cli/lib/soupify/run-entrypoint-file.ts]: *continuing after errors

File: cli/lib/soupify/run-entrypoint-file.ts [hunk 2]
```diff
@@ -17,44 +17,49 @@ export const runEntrypointFile = async (
...
-    ctx.runtime === "node"
-      ? $`npx tsx ${tmpEntrypointPath}`
-      : $`bun ${tmpEntrypointPath}`
-
-  debug(`starting process....`)
-  const processResult = await processCmdPart1
-    .stdout(debug.enabled ? "inheritPiped" : "piped")
-    .stderr(debug.enabled ? "inheritPiped" : "piped")
-    .noThrow()
+  try {
```
Comment (seveibar) [inline cli/lib/soupify/run-entrypoint-file.ts]: do we need this big try catch here?

File: cli/lib/soupify/soupify-with-core.ts [hunk 3]
```diff
@@ -22,52 +22,49 @@ export const soupifyWithCore = async (
...
-  writeFileSync("${tmpOutputPath}", JSON.stringify({
-    COMPILE_ERROR: e.toString() + "\\n\\n" + e.stack,
-  }))
-  throw e
-}
-
-
-writeFileSync("${tmpOutputPath}", JSON.stringify(project.getCircuitJson()))
-`.trim(),
+    import React from "react"
```
Comment (seveibar) [inline cli/lib/soupify/soupify-with-core.ts]: it's actually correct here to not have indentation, the output file is going to have a lot of indentation if we add indentation here

**Review by seveibar — CHANGES_REQUESTED** [review]
looking good, some minor tweaks and this is ready to ship!

File: cli/lib/soupify/run-entrypoint-file.ts (hunk 2 above)
Comment (ShiboSoftwareDev) [reply cli/lib/soupify/run-entrypoint-file.ts]: I forgot to remove these

File: cli/lib/soupify/soupify-with-core.ts (hunk 3 above)
Comment (ShiboSoftwareDev) [reply cli/lib/soupify/soupify-with-core.ts]: I'll reverse this change

File: cli/lib/export-fns/export-gerbers.ts [hunk 4]
```diff
@@ -29,6 +29,7 @@ export const exportGerbersToFile = async (
     },
     ctx,
   )
+  console.log(soup, "soup")
```
Comment (seveibar) [inline cli/lib/export-fns/export-gerbers.ts]:
```suggestion
```

File: cli/lib/export-fns/export-gerbers.ts [hunk 5]
```diff
@@ -91,7 +92,7 @@ export const exportGerbersToZipBuffer = async (
 ) => {
   const tempDir = Path.join(".tscircuit", "tmp-gerber-zip")
   fs.mkdirSync(tempDir, { recursive: true })
-
+  console.log("step two")
```
Comment (seveibar) [inline cli/lib/export-fns/export-gerbers.ts]:
```suggestion
```

File: cli/lib/export-fns/export-gerbers.ts [hunk 6]
```diff
@@ -100,6 +101,7 @@ export const exportGerbersToZipBuffer = async (
     },
     ctx,
   )
+  console.log("step last")
```
Comment (seveibar) [inline cli/lib/export-fns/export-gerbers.ts]:
```suggestion
```

File: cli/lib/soupify/run-entrypoint-file.ts [hunk 7]
```diff
@@ -22,12 +22,11 @@ export const runEntrypointFile = async (
...
       ? $`npx tsx ${tmpEntrypointPath}`
       : $`bun ${tmpEntrypointPath}`
-
   debug(`starting process....`)
+
   const processResult = await processCmdPart1
-    .stdout(debug.enabled ? "inheritPiped" : "piped")
     .stderr(debug.enabled ? "inheritPiped" : "piped")
-    .noThrow()
+    .stdout(debug.enabled ? "inheritPiped" : "piped")
```
Comment (seveibar) [inline cli/lib/soupify/run-entrypoint-file.ts:29]: Hmm why remove noThrow here?

**Review by seveibar — APPROVED** [review]

**Comment by ShiboSoftwareDev** [comment]
@seveibar I used to review my own code I don't why I stopped doing that, sorry for that

**Comment by seveibar** [comment]
@ShiboSoftwareDev all good, we should enable a lint rule for console log lines anyway so that its not easy to leave them in- i think biome has a setting

**Pull request merged**